faculty/clients/job.py

```diff
@@ -580,3 +580,19 @@ def get_subrun(
             project_id, job_id, run_identifier, subrun_identifier
         )
         return self._get(endpoint, SubrunSchema())
+
+    def cancel_run(self, project_id, job_id, run_identifier):
+        """Cancel a runnig job.
+
+        Parameters
+        ----------
+        project_id : uuid.UUID
+        job_id : uuid.UUID
+        run_identifier : uuid.UUID
```

Comment:
```suggestion
        run_identifier : uuid.UUID or int
```
If this method is going to support run numbers